### PR TITLE
[LC-287] Fixed the time delay until tx was confirmed

### PR DESCRIPTION
**Pull request by windies21**
Fixed the time delay until tx was confirmed

### CODE REVIEW

File: loopchain/peer/consensus_siever.py [hunk 1]
```diff
@@ -135,36 +157,34 @@ def __build_candidate_block(self, block_builder, next_leader, vote_result):
...
-    #             vote_result = await asyncio.wait_for(self._vote_queue.get(), timeout=timeout)
-    #             if vote_result is None:  # sentinel
-    #                 return False
-    #
-    #         except asyncio.TimeoutError:
-    #             logging.warning("Timed Out Block not confirmed duration: " +
-    #                             str(util.diff_in_seconds(candidate_block.header.timestamp)))
-    #             return False
+                if not conf.ALLOW_MAKE_EMPTY_BLOCK:
+                    self.__block_generation_timer.call_ignore_slot()
```
Comment (zsaladin) [inline loopchain/peer/consensus_siever.py]: How about changing the method name?
For example `call_instantly`, `call_immediately`

File: loopchain/peer/consensus_siever.py [hunk 2]
```diff
@@ -135,36 +157,34 @@ def __build_candidate_block(self, block_builder, next_leader, vote_result):
...
-    #
-    #             vote_result = await asyncio.wait_for(self._vote_queue.get(), timeout=timeout)
-    #             if vote_result is None:  # sentinel
-    #                 return False
-    #
-    #         except asyncio.TimeoutError:
-    #             logging.warning("Timed Out Block not confirmed duration: " +
-    #                             str(util.diff_in_seconds(candidate_block.header.timestamp)))
-    #             return False
+                if not conf.ALLOW_MAKE_EMPTY_BLOCK:
```
Comment (zsaladin) [inline loopchain/peer/consensus_siever.py]: This configuration name seems to be not correct.

File: loopchain/peer/consensus_siever.py (hunk 1 above)
Comment (zsaladin) [reply loopchain/peer/consensus_siever.py]: Or just call `await consensus()`

File: loopchain/peer/consensus_siever.py [hunk 3]
```diff
@@ -125,8 +145,10 @@ def __build_candidate_block(self, block_builder, next_leader, vote_result):
             self._blockchain.last_unconfirmed_block = candidate_block
             broadcast_func = partial(self._block_manager.broadcast_send_unconfirmed_block, candidate_block)
 
-            # TODO Temporary ignore below line for developing leader complain
             self.__start_broadcast_send_unconfirmed_block_timer(broadcast_func)
+            success = await self._wait_for_voting(candidate_block)
```
Comment (zsaladin) [inline loopchain/peer/consensus_siever.py]: Other voting points are not handled(line: 126). Is it okay?

File: loopchain/peer/consensus_siever.py (hunk 2 above)
Comment (windies21) [reply loopchain/peer/consensus_siever.py]: I agree with your opinion, but the configuration name is not made by this branch. We will consider it in another branch.

**Pull request merged**